*** ./smali/android/provider/Settings$System.smali	2015-09-03 07:17:20.297837681 +0800
--- /home/baohailong/patchrom/900901/temp/new_smali/framework.jar.out/./smali/android/provider/Settings$System.smali	2015-09-03 07:19:35.782780509 +0800
***************
*** 1527,1532 ****
--- 1527,1534 ----
      iput v0, p1, Landroid/content/res/Configuration;->fontScale:F
  
      :cond_0
+     invoke-static {p0, p1, p2}, Landroid/provider/MiuiSettings;->getConfigurationForUser(Landroid/content/ContentResolver;Landroid/content/res/Configuration;I)V
+ 
      return-void
  .end method
  
