*** ./smali/android/app/DialogFragment.smali	2015-09-03 07:17:46.286020955 +0800
--- /home/baohailong/patchrom/900901/temp/new_smali/framework.jar.out/./smali/android/app/DialogFragment.smali	2015-09-03 07:20:03.470969665 +0800
***************
*** 421,426 ****
--- 421,430 ----
      return-void
  
      :cond_1
+     iget-object v2, p0, Landroid/app/DialogFragment;->mDialog:Landroid/app/Dialog;
+ 
+     if-eqz v2, :cond_0
+ 
      invoke-virtual {p0}, Landroid/app/DialogFragment;->getView()Landroid/view/View;
  
      move-result-object v1
