*** ./smali/android/app/FragmentManagerImpl$6.smali	2015-09-03 07:17:51.058054476 +0800
--- /home/baohailong/patchrom/900901/temp/new_smali/framework.jar.out/./smali/android/app/FragmentManagerImpl$6.smali	2015-09-03 07:20:07.526997285 +0800
***************
*** 49,54 ****
--- 49,60 ----
  
      iget-object v0, p0, Landroid/app/FragmentManagerImpl$6;->val$finalFragment:Landroid/app/Fragment;
  
+     iget-boolean v0, v0, Landroid/app/Fragment;->mHidden:Z
+ 
+     if-eqz v0, :cond_0
+ 
+     iget-object v0, p0, Landroid/app/FragmentManagerImpl$6;->val$finalFragment:Landroid/app/Fragment;
+ 
      iget-object v0, v0, Landroid/app/Fragment;->mView:Landroid/view/View;
  
      const/16 v1, 0x8
